blog/2023-07-13-reproducibility-designer.md

```diff
@@ -0,0 +1,41 @@
+---
+author: Antonin Delpeuch
+title: "Designer seeked for OpenRefine's reproducibility improvements"
+---
+
+OpenRefine is hiring a designer to help shape our improvements to OpenRefine's operation history and test those changes with user panels.
+<!--truncate-->
+
+[OpenRefine](https://openrefine.org/) is a powerful tool to clean messy data, popular in a diverse range of communities. It has been serving the needs of journalists, librarians, Wikipedians, scientists for more than ten years and is taught in many curricula and workshops around the world. OpenRefine is a fiscally sponsored project of [Code for Science & Society Inc](https://codeforscience.org/), a 501(c)(3) charitable organization in the USA.
+
+We are currently working on improving the reproducibility of OpenRefine workflows as part of [a project funded by the Essential Open Source Software for Science programme](https://chanzuckerberg.com/eoss/proposals/improving-openrefines-reproducibility/).
```

Comment:
OpenRefine doesn't have "workflows." It has an Undo history which was inadvisably advertised as a pseudo-workflow-like capability without any of the design or engineering to make it actually work as a workflow.

I think it would be better to recast the job description in this light (ie it's a new design task, not "fixing" an old design)

Reply:
I've changed the wording here, can you have a look if it suits you?

Reply:
That looks better. I took a crack at editing it a little bit more.